Removing internal from ProfiledSqlDbCommand

MattLud opened the pull request:
I added a license, support for nhibernate 3.3.3.4001 and removed the internal for the sqlcommand class. 

The reasoning behind it comes from setting a custom type from the SQLCommand properties for a custom type - this code would not compile since the command class is not visible.  

```
public void NullSafeSet(IDbCommand st, object value, int index, ISessionImplementor session)
            {
                var s = st as SqlCommand;

                if(s == null)
                {
                    var sz = st as StackExchange.Profiling.NHibernate.Infrastructure.ProfiledSqlDbCommand;
                    s = sz.SqlCommand;
                    if (s == null)
                    {
                        throw new NotImplementedException();
                    }
                }
                s.Parameters[index].SqlDbType = SqlDbType.Structured;
                s.Parameters[index].TypeName = "StringList";
                s.Parameters[index].Value = value;
            }

```

MattLud committed:
Moving from internal to public to help with custom data types.

MattLud committed:
Adding License to be more inline with main miniprofiler

MattLud committed:
Adding support for NHibernate 3.3.3.4001

MattDavies commented:
Hi there,

Thanks for this pull request - awesome to see the library being used and I really appreciate the contribution :)

Can you just double check the change to the .csproj - I've added a comment. If you don't get a chance, I will take a look at this tonight and push out an updated package.

Good point on the license as well.

Thanks again!

Cheers
Matt

MattLud commented:
Sure thing. I'll do a quick fix this afternoon and update the request.

MattLud committed:
Fixing forgotten csproject reference

Had forgotten to build after adding - .csproj is not saved after nuget
package refs are added/removed

robdmoore commented:
Hey matt I think it should be the MIT license not Apache so its consistent with the other MRCollective projects - thoughts?

MattLud commented:
I think that'd be compatible. I went with a license that miniprofiler is using to keep things consistent.

MattLud commented:
Checking in on this again - anything else you need to accept the pull request?

MattDavies commented:
Done - sorry I didn't see the notification when you updated the PR :)

Thanks for the contribution, much appreciated!

MattDavies commented:
I'll get an updated nuget package out tomorrow as well.
 On 30/10/2013 11:53 PM, "Matt L." notifications@github.com wrote:

> Checking in on this again - anything else you need to accept the pull
> request?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/MRCollective/MiniProfiler.NHibernate/pull/2#issuecomment-27402663
> .
